web/src/components/storage/BootSelection.jsx

```diff
@@ -88,7 +87,7 @@ export default function BootSelectionDialog() {
 
       setState({
         load: true,
-        bootDevice: availableDevices.find(d => d.name === bootDevice),
+        bootDevice: availableDevices.find(d => d.name === bootDevice) || defaultBootDevice || availableDevices[0],
```

Comment:
Actually my proposed fix is wrong. Please, revert it. I will create a PR with some fixes for the default boot device.
```suggestion
        bootDevice: availableDevices.find(d => d.name === bootDevice) || availableDevices[0],
```